clients/egui/Cargo.toml

```diff
@@ -5,16 +5,26 @@ edition = "2021"
 
 [dependencies]
 dark-light = "1.0.0"
-eframe = "0.22.0"
-egui-notify= "0.7.0"
-egui-winit = "0.22.0"
-egui_extras = { version = "0.22.0", features = ["image"] }
-egui_wgpu_backend = "0.24.0"
-image = { version = "0.24", default-features = false, features = ["png", "jpeg", "gif", "webp", "bmp", "ico"] }
-lb = { package = "lb-rs", path = "../../libs/lb/lb-rs", default-features = false, features=["native-tls"] }
+egui = "0.26.0"
+eframe = "0.26.0"
+egui-notify = "0.13.0"
+egui-winit = "0.26.0"
+egui_extras = { version = "0.26.0", features = ["image"] }
+egui_wgpu_backend = { path = "../../../../egui_wgpu_backend" }
```

Comment:
todo: update after https://github.com/hasenbanck/egui_wgpu_backend/pull/80 is merged/released